src/posts/lucky_0.23.cr

```diff
@@ -0,0 +1,219 @@
+class Lucky023Release < BasePost
+  title "Lucky v0.23 is out, and it's huge!"
+  slug "lucky-0_23-release"
+
+  def published_at : Time
+    Time.utc(year: 2020, month: 6, day: 30)
+  end
+
+  def summary : String
+    <<-TEXT
+    Lucky v0.23 is out now. Lots of new features, bug fixes,
+    and even some performance boosts!
+    TEXT
+  end
+
+  def content : String
+    <<-MD
+    [Lucky v0.23](https://github.com/luckyframework/lucky/blob/master/CHANGELOG.md#changes-in-023)
+    is out now and works with the newest version of Crystal (v0.35.0)
+
+    Be sure to upgrade your version of Crystal, and take a look at our
+    [UPGRADE NOTES](https://github.com/luckyframework/lucky/blob/master/UPGRADE_NOTES.md#upgrading-from-022-to-023)
+    for help with migrating your app.
+
+    ### LuckyDiff
+
+    Thanks to a community member [@stephendolan](https://github.com/stephendolan/lucky_diff) for
+    creating this awesome tool, we have started linking to it in our UPGRADE_NOTES to help you
+    see what the differences are between versions!
+
+    ### What happened to 0.22.0?
+
+    Don't worry, you didn't miss anything. If you're using Lucky 0.21.0, you can upgrade to
+    [0.22.0](https://github.com/luckyframework/lucky/blob/master/CHANGELOG.md#v0220-2020-06-17) just by
+    updating your Crystal version to 0.35.0. This release didn't change anything other than compatibility
+    with the latest Crystal. To get all of the latest goodies, you'll want to go up to 0.23.0.
+
+    ## Quick Updates
+
+    ### `mount` is now `m`
+
+    We have deprecated the use of `mount` in favor of a shorter method `m`. The structure of this
+    is a little bit different.
+
+    ```crystal
+    # Before
+    mount MyComponent.new(arg: 1)
+
+    # After
+    m MyComponent, arg: 1
+    ```
+
+    As you can see, you still have all the flexibility, but in a bit more compact version!
+
+    ### Params
+
+    When you need to access a param value, you'll use `params.get(:key)`, but now, the blankspace
+    will be stripped automatically. If you need to get the untouched value, you can use `params.get_raw`.
```

Comment:
```suggestion
    Leading and trailing whitespace is now stripped automatically when accessing params. If you need to get the unstripped value, you can use `params.get_raw(:the_key)`.
```

And maybe an example like `For example if a user sends "email@example.com " Lucky will remove the trailing whitespace and return `"email@example.com"`